content/news/2016/11/2016-11-09-challenge-gov-launches-one-click-reporting-for-annual-prizes-data-call.md

```diff
@@ -2,7 +2,7 @@
 slug: challenge-gov-launches-one-click-reporting-for-annual-prizes-data-call
 date: 2016-11-09 5:43:18 -0400
 title: Challenge.gov Launches One-Click Reporting for Annual Prizes Data Call
-summary: 'This year, the deadline for agencies to submit their reporting of incentive prize competitions and challenges for FY16 comes earlier than most. Roughly two weeks from today, by Nov. 18, federal agencies are required to submit their accounts of every prize, competition, or challenge that launched, ran or completed in FY16 via email.  {{< legacy-img'
+summary: 'This year, the deadline for agencies to submit their reporting of incentive prize competitions and challenges for FY16 comes earlier than most. Roughly two weeks from today, by Nov. 18, federal agencies are required to submit their accounts of every prize, competition, or challenge that launched, ran or completed in FY16 via email.'
```

Comment:
The `{{< legacy-img'` was causing the build to fail locally.

Reply:
Probably shouldn't be there anyways